### PR TITLE
Update to Virannot 0.1.0

### PR DESCRIPTION
**Pull request by marieBvr**
FOR CONTRIBUTOR:
* [x] I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] License permits unrestricted use (educational + commercial)
* [ ] This PR adds a new tool or tool collection
* [x] This PR updates an existing tool or tool collection
* [ ] This PR does something else (explain below)

Fix BLASTXML parsing.
OTU module create a fasta file for each pfam motif.

### CODE REVIEW

File: tools/virAnnot/virAnnot_otu.xml [hunk 1]
```diff
@@ -73,9 +73,9 @@
     </outputs>
     <tests>
         <test>
-            <param name="blast_files" value="otu_s1_tblastx.tab,otu_s2_tblastx.tab"/>
```
Comment (bgruening) [inline tools/virAnnot/virAnnot_otu.xml:76]: can those old files be removed?

File: tools/virAnnot/otu.py [hunk 2]
```diff
@@ -355,7 +360,9 @@ def _get_stats(options, hits_collection):
         worksheet.write(row, column + 2, 'contigs_list')
         row = 1
         # column = 0
+        f_nucc = open(file_fasta_nucc, "w+")
```
Comment (bgruening) [inline tools/virAnnot/otu.py:363]: You can use `with open(file_fasta_nucc, "w+") as f_nucc` here. Its the new recommended way in python. This way you also don't need to close() the handle.

File: tools/virAnnot/virAnnot_otu.xml (hunk 1 above)
Comment (marieBvr) [reply tools/virAnnot/virAnnot_otu.xml:76]: You are right, it is done.

File: tools/virAnnot/otu.py (hunk 2 above)
Comment (marieBvr) [reply tools/virAnnot/otu.py]: The change has been made, for this line and another one.
Thanks.

**Comment by bgruening** [comment]
Thanks @marieBvr

**Pull request merged**